tests/test_autoresize.lua

```diff
@@ -279,4 +279,116 @@ T['autoresize']['maximize_minwidth'] = function()
     validate_win_dims(win_id_right, { 12, 23 })
 end
 
+T['autoresize']['resize when switching modes (vsplit)'] = function()
+    reload_module({})
+    edit(lorem_ipsum_file)
+
+    child.cmd('vsplit')
+    child.cmd('FocusMaximise')
+
+    local resize_state = child.get_resize_state()
+    local win_id_left = resize_state.windows[1]
+    local win_id_right = resize_state.windows[2]
+
+    -- we should be in the left win
+    eq(win_id_left, child.api.nvim_get_current_win())
+
+    -- let should take up the entire width, other than the winseparator (1 col)
```

Comment:
What should take up the entire width?

Reply:
Oops, that's just a comment left from copy-pasting. I'll clean those up.